src/appUrls/index.js

```diff
@@ -7,10 +7,12 @@ export const API_URLS = {
   FETCH_PROFILE: `${BASE_URL}/api/profiles/`,
   EDIT_PROFILE: `${BASE_URL}/api/profiles/`,
   FETCH_BOOKMARKS: `${BASE_URL}/api/bookmarks/`,
-  CREATE_BOOKMARK: `${BASE_URL}/api/articles/{}/bookmark/`,
-  DELETE_BOOKMARK: `${BASE_URL}/api/articles/{}/unbookmark/`,
   PUBLISHED_ARTICLES:`${BASE_URL}/api/me/articles/published/`,
-  DRAFTED_ARTICLES:`${BASE_URL}/api/me/articles/drafts/`
+  DRAFTED_ARTICLES:`${BASE_URL}/api/me/articles/drafts/`,
+  CREATE_BOOKMARK: `${BASE_URL}/api/articles/`,
+  DELETE_BOOKMARK: `${BASE_URL}/api/articles/`
 };
 
 export const LOGIN_URL = `${BASE_URL}/api/users/login/`;
+
+export const svg_url = "M19.385 4h-9.77A2.623 2.623 0 0 0 7 6.615V23.01a1.022 1.022 0 0 0 1.595.847l5.905-4.004 5.905 4.004A1.022 1.022 0 0 0 22 23.011V6.62A2.625 2.625 0 0 0 19.385 4zM21 23l-5.91-3.955-.148-.107a.751.751 0 0 0-.884 0l-.147.107L8 23V6.615C8 5.725 8.725 5 9.615 5h9.77C20.275 5 21 5.725 21 6.615V23z";
```

Comment:
This svg_url should be in the assets folder i.e `src/assets/articleAssets/svgIcons.js`

Reply:
Changed